src/modules/noit_lua_noit_binding.c

```diff
@@ -106,214 +105,6 @@ lua_general_filtersets_cull(lua_State *L) {
 }
 
 static int
-lua_noit_metric_adjustsubscribe(lua_State *L, short bump) {
```

Comment:
missing file... i include the other C file that was missing that has these definitions.